homeassistant/components/ebusd/const.py

```diff
@@ -77,23 +77,23 @@
         ],
     },
     "ehp": {
-        "HWTemperature": ["HwcTemp", "°C", "mdi:thermometer", 4],
-        "OutsideTemp": ["OutsideTemp", "°C", "mdi:thermometer", 4],
+        "HWTemperature": ["HwcTemp", TEMP_CELSIUS, "mdi:thermometer", 4],
+        "OutsideTemp": ["OutsideTemp", TEMP_CELSIUS, "mdi:thermometer", 4],
     },
     "bai": {
-        "HotWaterTemperature": ["HwcTemp", "°C", "mdi:thermometer", 4],
-        "StorageTemperature": ["StorageTemp", "°C", "mdi:thermometer", 4],
-        "DesiredStorageTemperature": ["StorageTempDesired", "°C", "mdi:thermometer", 0],
-        "OutdoorsTemperature": ["OutdoorstempSensor", "°C", "mdi:thermometer", 4],
-        "WaterPreasure": ["WaterPressure", "bar", "mdi:pipe", 4],
-        "AverageIgnitionTime": ["averageIgnitiontime", "seconds", "mdi:av-timer", 0],
-        "MaximumIgnitionTime": ["maxIgnitiontime", "seconds", "mdi:av-timer", 0],
-        "MinimumIgnitionTime": ["minIgnitiontime", "seconds", "mdi:av-timer", 0],
-        "ReturnTemperature": ["ReturnTemp", "°C", "mdi:thermometer", 4],
+        "HotWaterTemperature": ["HwcTemp", TEMP_CELSIUS, "mdi:thermometer", 4],
+        "StorageTemperature": ["StorageTemp", TEMP_CELSIUS, "mdi:thermometer", 4],
+        "DesiredStorageTemperature": ["StorageTempDesired", TEMP_CELSIUS, "mdi:thermometer", 0],
+        "OutdoorsTemperature": ["OutdoorstempSensor", TEMP_CELSIUS, "mdi:thermometer", 4],
+        "WaterPreasure": ["WaterPressure", PRESSURE_BAR, "mdi:pipe", 4],
+        "AverageIgnitionTime": ["averageIgnitiontime", ATTR_SECONDS, "mdi:av-timer", 0],
```

Comment:
We only use `ATTR_*` constants in service attribute keys.

Reply:
Does it mean I have to remove ATTR_* !?
Done it :)